dashboard/modules/metrics/dashboards/serve_deployment_dashboard_panels.py

```diff
@@ -149,6 +149,108 @@
         stack=False,
         grid_pos=GridPos(16, 2, 8, 8),
     ),
+    Panel(
```

Comment:
for most users these panels will be useless.

Is it possible to put them behind a separate grafana dashboard? or maybe a collapsed-by-default grouping?

Reply:
@edoakes I talked to @alanwguo, `row` is not supported in our dashboard, i think I can take it as a follow up to support it. 
issue: https://github.com/ray-project/ray/issues/38024